ios/engine/KMEI/KeymanEngine/Classes/Keyboard/InputViewController.swift

```diff
@@ -408,6 +408,10 @@ open class InputViewController: UIInputViewController, KeymanWebDelegate {
   func updateShowBannerSetting() {
     keymanWeb.updateShowBannerSetting()
   }
+  
+  func updateSpacebarText() {
+    keymanWeb.updateSpacebarText()
+  }
```

Comment:
I'd prefer this (and the `KeymanWebViewController` func) to be properties, not methods.  Just because _we_ use the setting globally doesn't mean everyone will want it be be global.

Just like KMW's doing an OSK-core thing that will allow swappable OSKs, I've been trying to ensure that any changes made to the iOS keyboard have a similar mindset behind them.

Reply:
Can you elaborate? This makes no sense to me. It's a global setting; how would you apply it otherwise?

Remember also that I am trying to match the APIs between iOS and Android here.

Reply:
I'm not saying to not have the global setting; I'm fine with having it for those who want to treat it as global through `Manager`.  I'm saying to not treat it as global within instances of the keyboard, in case someone wants to set it on specific instances of `InputViewController`.

Reply:
Okay ... seems overly complicated to me and I can't see a use case but whatever.

FWIW (unrelated), I noted multiple instances of the web view in Safari inspector when testing this in the System Keyboard -- instead of just 1, I'd see up to seven or eight. Only one was active but smells like a leak somewhere...